tests/tasks/verify_settings.yml

```diff
@@ -104,21 +104,40 @@
           __verify_mssql_agent_is_enabled | string | lower in
           __mssql_conf_get_sqlagent.stdout | lower
 
-- name: Verify the mssql-server-fts package
+- name: Verify the {{ __mssql_server_fts_packages }} package
```

Comment:
It's generally not good to have unrelated changes in a PR - it would be better to have these sort of "clean up" commits in a separate PR

Reply:
Got you, I've moved the unrelated changes to  #21.